jerry-ext/include/jerryscript-ext/arg.h

```diff
@@ -64,6 +64,12 @@ jerry_value_t jerryx_arg_transform_args (const jerry_value_t *js_arg_p,
                                          const jerryx_arg_t *c_arg_p,
                                          jerry_length_t c_arg_cnt);
 
+jerry_value_t jerryx_arg_transform_object_properties (const jerry_value_t taregt,
+                                                      const jerry_char_t **name_p,
+                                                      const jerry_length_t name_cnt,
+                                                      const jerryx_arg_t *c_arg_p,
+                                                      jerry_length_t c_arg_cnt);
```

Comment:
> Do we need two counters?

Yes, just like in `jerryx_arg_transform_args`, we have both js_arg_cnt and c_arg_cnt.
Because name_p and c_arg_p are not 1:1 mappings (depends on the numbers of `jerryx_arg_js_iterator_pop` and `jerryx_arg_js_iterator_peek` used in each validator function).

> Why one of them is const and the other is not?

because `c_arg_cnt`'s value will change in the function call. see line41 in `jerryx_arg_transform_args`: we have `c_arg_cnt--`

> const is not needed for non pointer types.

aligned with other functions' signature in arg.c. And also I think it is a good habit to explicitly declare some vars as const, so that some IDE can help us find the error if we changed its value by mistake, right?